src/core/internal/array/appending.d

```diff
@@ -149,14 +157,26 @@ private void __doPostblit(T)(auto ref T t) @trusted pure
         static if (__traits(hasMember, T, "__xpostblit") &&
                 // Bugzilla 14746: Check that it's the exact member of S.
                 __traits(isSame, T, __traits(parent, t.__xpostblit)))
-            t.__xpostblit();
+        {
+            import core.internal.array.utils : isPure;
+            static if (isPure!(t.__xpostblit))
```

Comment:
Should this be `static if (isPostblitPure!T)`?  I don't see `isPostblitPure` used anywhere.